### PR TITLE
update quickstart test

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by Le-Zheng** [comment]
http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Quickstart-notebook/241/

File: docs/docs/colab-notebook/run-notebook-tf2.sh [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 now=$(date "+%s")
 time1=$((now - start))
 
 echo "#2 start test for ncf_dataframe.ipynb"
 start=$(date "+%s")
-sed -i 's/\!/#/g' ${ANALYTICS_ZOO_HOME}/docs/docs/colab-notebook/orca/quickstart/ncf_dataframe.ipynb
-ipython ${ANALYTICS_ZOO_HOME}/docs/docs/colab-notebook/orca/quickstart/ncf_dataframe.ipynb
+${ANALYTICS_ZOO_HOME}/apps/ipynb2py.sh ${ANALYTICS_ZOO_HOME}/docs/docs/colab-notebook/orca/quickstart/ncf_dataframe
+sed -i "s/get_ipython()/#/g" ${ANALYTICS_ZOO_HOME}/docs/docs/colab-notebook/orca/quickstart/ncf_dataframe.py
+python ${ANALYTICS_ZOO_HOME}/docs/docs/colab-notebook/orca/quickstart/ncf_dataframe.py
```
Comment (yushan111) [inline docs/docs/colab-notebook/run-notebook-tf2.sh:16]: We may also need to add exit status check.
```bash 
exit_status=$?
if [ $exit_status -ne 0 ];
then
    clear_up
    echo "xxx failed"
    exit $exit_status
fi
```

Comment (Le-Zheng) [reply docs/docs/colab-notebook/run-notebook-tf2.sh:32]: Updated

**Comment by Le-Zheng** [comment]
http://10.239.47.210:18888/job/ZOO-PR-Quickstart-notebook/244/ jenkins passed

**Review by yushan111 — APPROVED** [review]

**Comment by Le-Zheng** [comment]
jenkins passd http://10.239.47.210:18888/job/ZOO-PR-Quickstart-notebook/

**Pull request merged**